src/viewport.js

```diff
@@ -970,9 +987,11 @@ $.Viewport.prototype = {
          * @type {object}
          * @property {OpenSeadragon.Viewer} eventSource - A reference to the Viewer which raised the event.
          * @property {Number} degrees - The number of degrees the rotation was set to.
+         * @property {Boolean} immediately - Whether the rotation happened immediately or was animated
```

Comment:
Interesting idea to add this... I guess I don't see why not!

Reply:
I thought it would be useful for event subscribers to be able know whether to apply changes immediately, for example when synchronizing overlays or such. Doesn't seem like it would hurt anything, at the very least.

Reply:
Agreed!